fetch_pyqt_windows.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I see. In that case let's leave the .cfg in-place for now since that sounds like a good reason to keep it there.